systemtest/src/main/java/io/strimzi/systemtest/storage/TestStorage.java

```diff
@@ -72,15 +76,19 @@ public TestStorage(ExtensionContext extensionContext, String namespaceName, int
         this.namespaceName = StUtils.isParallelNamespaceTest(extensionContext) ? StUtils.getNamespaceBasedOnRbac(namespaceName, extensionContext) : namespaceName;
         this.clusterName = CLUSTER_NAME_PREFIX + hashStub(String.valueOf(RANDOM.nextInt(Integer.MAX_VALUE)));
         this.kafkaNodePoolName = Constants.KAFKA_NODE_POOL_PREFIX + hashStub(clusterName);
+        this.sourceClusterName = clusterName + "-source";
         this.targetClusterName = clusterName + "-target";
         this.topicName = KafkaTopicUtils.generateRandomNameOfTopic();
+        this.sourceTopicName = topicName + "-source";
```

Comment:
looking on how mm2 test are refactored i think this can be a bit tricky, it is refactored in such a manner that we are producing something to source cluster topic called either `testStorage.getTopicName` or `testStorage.getSourceTopicName` and latter expect it to be  mirrored to the topic called `testStorage.getTargetTopicName` ? 
the logic I am missing here is maping like `kafkaClusterSourceName + "." + testStorage.getTopicName();` when trying to obtain name of mirrored topic in target cluster.

Reply:
You are absolutely right! I confused the logic there. Changing that now.